documentation/how-to/cli.md

```diff
@@ -0,0 +1,38 @@
+# Temporal CLI
+
+[temporal cli](https://docs.temporal.io/cli) is a command-line tool that can be used
```

Comment:
s/cli/CLI?

In every instance where this applies.

Reply:
addressed in [2602479](https://github.com/canonical/temporal-k8s-operator/pull/82/commits/26024790314a08bf991333d30062fd4f4c386ae4)